projects/components/src/combo-box/combo-box.component.ts

```diff
@@ -135,6 +137,12 @@ export class ComboBoxComponent<TValue = string> implements AfterViewInit, OnChan
   @Output()
   public readonly escape: EventEmitter<void> = new EventEmitter();
 
+  @Output()
+  public readonly focused: EventEmitter<void> = new EventEmitter();
```

Comment:
This should emit true/false depending upon what the current state is. It should also emit when the focus and/or blur is cleared out from the component.

Reply:
Ah, agreed. that could be a good improvement. Let me change it.

Reply:
Hmm. On second thought, while I like that this reduces the API surface area it kind of feels semantically incorrect. Separating focus/blur is very explicit and matches the normal expectations for dom events. If we combine them the API would be suggesting that "not focused" is the same as "blurred" which isn't quite right.

Reply:
We shouldn't really be mimicing DOM events like this though, should we? Why not use the native events (focusin/out)?

Reply:
I didn't mean to combine them. I was thinking more like

```
   public readonly focused: EventEmitter<boolean> = new EventEmitter(); 

   public readonly blurred: EventEmitter<boolean> = new EventEmitter(); 
```